web/packages/teleterm/src/sharedProcess/ptyHost/ptyHostService.ts

```diff
@@ -73,5 +76,10 @@ export function createPtyHostService(): IPtyHost {
         });
     },
     exchangeEvents: stream => new PtyEventsStreamHandler(stream, ptyProcesses),
+    dispose: async () => {
+      await Promise.all(
+        Array.from(ptyProcesses).map(([, ptyProcess]) => ptyProcess.dispose())
```

Comment:
Nit: what about something like this?

```suggestion
        Array.from(ptyProcesses.values()).map(ptyProcess => ptyProcess.dispose())
```

Less unnecessary arrays get allocated and you don't need to destructure function args.